presto-product-tests/bin/run_on_docker.sh

```diff
@@ -76,18 +76,6 @@ function cleanup() {
   if [[ "${LEAVE_CONTAINERS_ALIVE_ON_EXIT:-false}" != "true" ]]; then
     stop_docker_compose_containers ${ENVIRONMENT}
   fi
-
-  # Ensure that the logs processes are terminated.
-  # In most cases after the docker containers are stopped, logs processes must be terminated.
-  # However when the `LEAVE_CONTAINERS_ALIVE_ON_EXIT` is set, docker containers are not being terminated.
-  # Redirection of system error is supposed to hide the `process does not exist` and `process terminated` messages
-  if test ! -z ${DOCKER_COMPOSE_LOGS_PID:-}; then
-    kill ${DOCKER_COMPOSE_LOGS_PID} 2>/dev/null || true
-  fi
-
-  # docker logs processes are being terminated as soon as docker container are stopped
-  # wait for docker logs termination
-  wait 2>/dev/null || true
```

Comment:
I think we still need to wait for docker-compose to finish

Reply:
i removed this based on the code comment, but i think you're right
also, since this is `|| true`, it's safe to do

Reply:
The best would be to remove `|| true` . But this is out of the scope.